waspc/cli/src/Wasp/Cli/Command/Telemetry/Common.hs

```diff
@@ -2,17 +2,14 @@ module Wasp.Cli.Command.Telemetry.Common
   ( TelemetryCacheDir,
     ensureTelemetryCacheDirExists,
     getTelemetryCacheDirPath,
+    getWaspCacheDirPath,
   )
 where
 
 import StrongPath (Abs, Dir, Path', reldir)
 import qualified StrongPath as SP
 import qualified System.Directory as SD
-
-data UserCacheDir
-
-getUserCacheDirPath :: IO (Path' Abs (Dir UserCacheDir))
-getUserCacheDirPath = SD.getXdgDirectory SD.XdgCache "" >>= SP.parseAbsDir
+import Wasp.Cli.Command.Utils.File (UserCacheDir, getUserCacheDirPath)
```

Comment:
We generally prefer to list symbols in a file from top to bottom: https://github.com/wasp-lang/wasp/pull/500#discussion_r824602425.

So when we encounter a file where this isn't the case, we shuffle them around a bit. Nothing to do here, this file is already in order, just FYI for future references.

Reply:
Good one! It is a typical Clean Code recommendation. I wonder if we should put that into out Coding Style guidelines.